README.md

```diff
@@ -78,6 +78,7 @@ in remote mode (meaning that `lnd-mode=remote` is set). It shows the
 
```

Comment:
Just leaving a note that this commit is not the last commit in the PR. I understand that this is intended to be fixed later though :)